openmls/src/framing/plaintext.rs

```diff
@@ -222,7 +226,9 @@ impl MlsPlaintext {
 
     /// This constructor builds an `MlsPlaintext` containing a Commit. The given
     /// `CommitType` determines the `SenderType`: If it's a `Member` commit,
-    /// it's `SenderType::Member` and `SenderType::NewMember` otherwise.
+    /// it's `SenderType::Member` and `SenderType::NewMember` otherwise. If it
+    /// is an `External` commit, the context is not signed along with the rest
```

Comment:
So `SenderType` can not only be `Member` and `NewMember`? This comment is confusing.